samples/server/petstore/fsharp-giraffe/src/OpenAPI.fsproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <Description>OpenAPI</Description>
+    <Copyright>OpenAPI</Copyright>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <DebugType>portable</DebugType>
+    <EnableDefaultContentItems>false</EnableDefaultContentItems>
+    <RunWorkingDirectory>$(MSBuildThisFileDirectory)</RunWorkingDirectory>
+    <AssemblyName>OpenAPI</AssemblyName>
+    <PackageId>OpenAPI</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" />
+    <PackageReference Include="Giraffe" Version="3.4.*" />
+    <PackageReference Include="OpenAPITypeProvider" Version="2.3.0" />
+    <PackageReference Include="TaskBuilder.fs" Version="2.1.*" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="openapi.yaml"/>
+    <Compile Include="Model.fs" />
+    <Compile Include="api/params/UrlParams.fs" />
+    <Compile Include="api/params/QueryParams.fs" />
+    <Compile Include="api/params/BodyParams.fs" />
+    <Compile Include="api/params/HeaderParams.fs" />

```

Comment:
@nmfisher Looks like these 4 files under `api/params` are not yet added to the project (templates)

Reply:
Actually that directory (samples/server/petstore/fsharp-giraffe/src) was the sample output directory from an earlier revision.

The proper output directory is samples/server/petstore/fsharp-giraffe/{{packageName}} (e.g. samples/server/petstore/fsharp-giraffe/OpenAPI)

I just forgot to remove the earlier version (samples/server/petstore/fsharp-giraffe/src) from the final commit. This is OK to delete - see https://github.com/nmfisher/openapi-generator/commit/ee4672ecf1fc6060e351797fe692736da4a4fc01